### PR TITLE
[Add][11.0] project_task_material_stock_with_sale_timesheet compatibility

### PR DESCRIPTION
**Pull request by jcarlosmontoya**
This module implements compatibility between
sale_timesheet module and project_task_material_stock to create Analytic Account Entries.

This module Fix issue Zero amount in Analytic Account Entries from Tasks with Materials
(When you use sale_timesheet module recompute amount to create Analytic lines always based in employee cost, so return zero amount on create or write, see image)
imagen

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@cubells can you check this to see if suitable?

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Last message deleted. Not related with topic.

**Comment by pedrobaeza** [comment]
@HaraldPanten this module is not for solving that things... It's for the second thing @jcarlosmontoya said in the issue (not related with the main topic by the way).

**Comment by jcarlosmontoya** [comment]
@HaraldPanten this module has other purpose, we have to create other pr I think

**Comment by HaraldPanten** [comment]
My apologies,

I misunderstood it's purpose.

**Review by HaraldPanten — APPROVED** [review]
Functional review

File: project_task_material_with_sale_timesheet/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "website": "https://github.com/OCA/project/",
+    "author": "Tecnativa, Praxya, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "project_task_material_stock",
+        "sale_timesheet",
+    ],
+    "data": [],
```
Comment (nikul-serpentcs) [inline project_task_material_with_sale_timesheet/__manifest__.py]: @jcarlosmontoya Not any File then remove it

**Review by nikul-serpentcs — COMMENTED** [review]
Minor change

File: project_task_material_with_sale_timesheet/__manifest__.py (hunk 1 above)
Comment (jcarlosmontoya) [reply project_task_material_with_sale_timesheet/__manifest__.py]: @nikul-serpentcs Thank you, data item?

**Comment by jcarlosmontoya** [comment]
@HaraldPanten when Could you do next PR?
Would be nice to create Stock picking instead Stock Moves for module project_task_material_stock
Best regards

File: project_task_material_with_sale_timesheet/README.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/11.0/project_task_material_stock
```
Comment (cubells) [inline project_task_material_with_sale_timesheet/README.rst]: As this is a fake README not created from OCA fragments, this url is incorrect. Can you create oca fragments?

File: project_task_material_with_sale_timesheet/README.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-11-0/project-11-0-project_task_material_stock
```
Comment (cubells) [inline project_task_material_with_sale_timesheet/README.rst]: As this is a fake README not created from OCA fragments, this url is incorrect. Can you create oca fragments?

**Review by cubells — CHANGES_REQUESTED** [review]
Ready to approve.

**Comment by jcarlosmontoya** [comment]
Hello Everyone, @cubells @rafaelbn Do we need to do anything else?
We need this fix please, This PR is checked ready for approve

File: project_task_material_with_sale_timesheet/README.rst (hunk 2 above)
Comment (rafaelbn) [reply project_task_material_with_sale_timesheet/README.rst]: Fix

File: project_task_material_with_sale_timesheet/README.rst (hunk 3 above)
Comment (rafaelbn) [reply project_task_material_with_sale_timesheet/README.rst]: Fix

**Comment by rafaelbn** [comment]
Hi @jcarlosmontoya , review please comments of @cubells  and ask @gelo-landoo to rewiew as he reported an issues with this problem

**Comment by jcarlosmontoya** [comment]
@rafaelbn thank you,
@cubells already approved it, @gelo-landoo Can you review it please?

**Comment by pedrobaeza** [comment]
@jcarlosmontoya yes, but with that remarks, so please honor them.

**Comment by jcarlosmontoya** [comment]
@pedrobaeza thank you, sure i'm checking it

File: project_task_material_with_sale_timesheet/README.rst (hunk 2 above)
Comment (jcarlosmontoya) [reply project_task_material_with_sale_timesheet/README.rst]: Updated ;)

File: project_task_material_with_sale_timesheet/README.rst (hunk 3 above)
Comment (jcarlosmontoya) [reply project_task_material_with_sale_timesheet/README.rst]: Updated

**Review by jcarlosmontoya — COMMENTED** [review]
Updated file

**Review by rafaelbn — APPROVED** [review]

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Comment by ghost** [comment]
@jcarlosmontoya sorry, I have just seen your mention today and i have realized the validations are done.
Regards,

**Pull request merged**